DSTV.Net/Extensions/TextReaderExtensions.cs

```diff
@@ -96,6 +96,7 @@ internal static async Task<int> ParseInteger(this TextReader reader, ReaderConte
     /// <summary>
     ///     Parses an enum of type <see cref="TEnum" />
     /// </summary>
+    /// <typeparam name="TEnum"></typeparam>
```

Comment:
Als we m toevoegen, moeten we m eigenlijk ook vullen met wat zinnige informatie. Wat is de betekening van TEnum hier?